### PR TITLE
Copy-editing docs

### PR DESCRIPTION
**Pull request by kgshv**
This PR contains just the copy-editing of the `README.md` file with some suggestions on text style.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+Current version is tested with Teamcity version **2018.1.5 (build 58744)**. Teamcity client can also be used with Java but will require **scala.collection.JavaConverters** when using collections (see examples below).
 
-TeamCity API collection entities conain a count property and a list property, The list property name is in single not plural, In order to stay as close to the API as possible, Teamcity client follows this convention. 
+Since Teamcity API only returns a small set of properties for each entity when getting a collection of entities, so we mapped these to `Base{Entity}`. You can use the `id` returned in `Base{Entity}` to retrieve the full properties of any select entity (by calling `get{Entity}ById` methods).
 
-Please note that not all the API calls and objects are covered in this TeamCityClient.
+The collection entities of TeamCity API contain a count property and a list property, and the list property name is in singular, not plural form - in order to stay as close to the API as possible, Teamcity client also follows this convention.
+
+Please note that not all API calls and objects are covered in this README.
```
Comment (uribwix) [inline README.md]: not in the readme but in the client, there are some apis and entities which we didnt cover in the client

Comment (kgshv) [reply README.md]: So the client does not use / wrap around every single TeamCity API endpoint, just some of the most important ones (or something like that), correct?

Comment (uribwix) [reply README.md]: correct, i wouldnt add the important ons as this is subjective to the application using the client

Comment (kgshv) [reply README.md]: Should be resolved now, please check.

**Comment by kgshv** [comment]
Hey @uribwix! Just checking to see if  there was anything else needed from to to merge this? Thanks!

**Pull request merged**